keras_cv/metrics/coco/recall.py

```diff
@@ -12,11 +12,173 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import tensorflow as tf
+import tensorflow.keras as keras
+import tensorflow.keras.initializers as initializers
 
-from keras_cv.metrics.coco.base import COCOBase
+from keras_cv.metrics.coco import iou as iou_lib
```

Comment:
For IOU, we might want to move it out of coco name space later, since this is not only applicable for coco metric.